### ISSUE TITLE
[Bug]: Crash related to Immersive Weathering and werewolves:stone_altar_fire_bowl

### ISSUE THREAD

**Issue by MaxWasUnavailable** [TeamLapen/Werewolves#101]
### Minecraft Version

1.18.x

### Forge Version

40.1.20

### Vampirism Version

Vampirism-1.18.2-1.8.2

### Werewolves Version

Werewolves-1.18.2-1.8-0.6.0-beta.3

### Singleplayer/Multiplayer

Singleplayer

### What happened?

Unknown. This was reported to me by a player who was testing my modpack. I'm assuming they were either walking about, or had placed an altar.

Update:

Tested this myself. I placed an altar fire on the ground while it was raining, and then waited ~5-10 minutes. I'm assuming Immersive Weathering detected the altar as something to be weathered (possibly some kind of bricks tags or such?), and tried to do so. The game crashed giving the same crash output.

### Relevant log output

```bash
https://pastebin.com/E6cTR2u0
```

### Reproduce steps

1. Place Stone Altar Fire.
2. Light the fire (might be optional?).
3. Wait.
4. Crash.

### Other relevant mods

Immersive Weathering: immersive_weathering-1.18.2-1.2.8.jar

**Comment by Cheaterpaul** [comment]
There is a problem in Immersive Weathering that assumes a the stone altar (fire bowls) are signal light.

I make a workaround.

Thanks for the detailed report

**Comment by MaxWasUnavailable** [comment]
It seems there's still an issue related to this. @Cheaterpaul 

Here is the relevant portion of the crash log of our dedicated server:

```
Description: Exception ticking world

java.lang.IllegalArgumentException: Cannot get property BooleanProperty{name=signal_fire, clazz=class java.lang.Boolean, values=[true, false]} as it does not exist in Block{werewolves:stone_altar_fire_bowl}
        at net.minecraft.world.level.block.state.StateHolder.m_61143_(StateHolder.java:98) ~[server-1.18.2-20220404.173914-srg.jar%23294!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:ferritecore.fastmap.mixin.json:FastMapStateHolderMixin,pl:mixin:A}
        at com.ordana.immersive_weathering.block_growth.hardcoded.CampfireGrowth.tryGrowing(CampfireGrowth.java:35) ~[immersive_weathering-1.18.2-1.2.10.jar%23168!/:1.18.2-1.2.10] {re:classloading}
        at com.ordana.immersive_weathering.block_growth.BlockGrowthHandler.tickBlock(BlockGrowthHandler.java:60) ~[immersive_weathering-1.18.2-1.2.10.jar%23168!/:1.18.2-1.2.10] {re:mixin,re:classloading}
        at net.minecraft.server.level.ServerLevel.handler$zmn000$callTick(ServerLevel.java:4237) ~[server-1.18.2-20220404.173914-srg.jar%23294!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:forgeendertech:coremod,xf:fml:sereneseasons:temperature_transformer,re:classloading,pl:accesstransformer:B,xf:fml:forgeendertech:coremod,xf:fml:sereneseasons:temperature_transformer,pl:mixin:APP:alternate_current.mixins.json:ServerLevelMixin,pl:mixin:APP:biomemakeover-common.mixins.json:multipart.ServerLevelMixin,pl:mixin:APP:blueprint.mixins.json:ServerLevelMixin,pl:mixin:APP:crafttweaker.mixins.json:common.transform.world.level.MixinServerLevel,pl:mixin:APP:enhancedcelestials.mixins.json:MixinServerWorld,pl:mixin:APP:immersive_weathering.mixins.json:ServerLevellMixin,pl:mixin:APP:immersive_weathering.mixins.json:ServerLevelSnowMixin,pl:mixin:APP:immersiveengineering.mixins.json:coremods.ServerWorldMixin,pl:mixin:APP:radium.mixins.json:ai.nearby_entity_tracking.ServerWorldAccessor,pl:mixin:APP:radium.mixins.json:alloc.chunk_random.ServerWorldMixin,pl:mixin:APP:radium.mixins.json:chunk.entity_class_groups.ServerWorldAccessor,pl:mixin:APP:radium.mixins.json:entity.inactive_navigations.ServerWorldMixin,pl:mixin:APP:radium.mixins.json:profiler.ServerWorldMixin,pl:mixin:APP:snowrealmagic.mixins.json:ServerLevelMixin,pl:mixin:APP:starlight.mixins.json:common.world.ServerWorldMixin,pl:mixin:APP:supplementaries.mixins.json:ServerLevelMixin,pl:mixin:A}
        at net.minecraft.server.level.ServerLevel.m_8714_(ServerLevel.java:443) ~[server-1.18.2-20220404.173914-srg.jar%23294!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:forgeendertech:coremod,xf:fml:sereneseasons:temperature_transformer,re:classloading,pl:accesstransformer:B,xf:fml:forgeendertech:coremod,xf:fml:sereneseasons:temperature_transformer,pl:mixin:APP:alternate_current.mixins.json:ServerLevelMixin,pl:mixin:APP:biomemakeover-common.mixins.json:multipart.ServerLevelMixin,pl:mixin:APP:blueprint.mixins.json:ServerLevelMixin,pl:mixin:APP:crafttweaker.mixins.json:common.transform.world.level.MixinServerLevel,pl:mixin:APP:enhancedcelestials.mixins.json:MixinServerWorld,pl:mixin:APP:immersive_weathering.mixins.json:ServerLevellMixin,pl:mixin:APP:immersive_weathering.mixins.json:ServerLevelSnowMixin,pl:mixin:APP:immersiveengineering.mixins.json:coremods.ServerWorldMixin,pl:mixin:APP:radium.mixins.json:ai.nearby_entity_tracking.ServerWorldAccessor,pl:mixin:APP:radium.mixins.json:alloc.chunk_random.ServerWorldMixin,pl:mixin:APP:radium.mixins.json:chunk.entity_class_groups.ServerWorldAccessor,pl:mixin:APP:radium.mixins.json:entity.inactive_navigations.ServerWorldMixin,pl:mixin:APP:radium.mixins.json:profiler.ServerWorldMixin,pl:mixin:APP:snowrealmagic.mixins.json:ServerLevelMixin,pl:mixin:APP:starlight.mixins.json:common.world.ServerWorldMixin,pl:mixin:APP:supplementaries.mixins.json:ServerLevelMixin,pl:mixin:A}
        at net.minecraft.server.level.ServerChunkCache.m_8490_(ServerChunkCache.java:359) ~[server-1.18.2-20220404.173914-srg.jar%23294!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:radium.mixins.json:alloc.chunk_ticking.ServerChunkManagerMixin,pl:mixin:APP:radium.mixins.json:world.chunk_access.ServerChunkManagerMixin,pl:mixin:APP:smoothchunk.mixins.json:ServerChunkCacheMixin,pl:mixin:A}
        at net.minecraft.server.level.ServerChunkCache.m_201698_(ServerChunkCache.java:309) ~[server-1.18.2-20220404.173914-srg.jar%23294!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:radium.mixins.json:alloc.chunk_ticking.ServerChunkManagerMixin,pl:mixin:APP:radium.mixins.json:world.chunk_access.ServerChunkManagerMixin,pl:mixin:APP:smoothchunk.mixins.json:ServerChunkCacheMixin,pl:mixin:A}
        at net.minecraft.server.level.ServerLevel.m_8793_(ServerLevel.java:281) ~[server-1.18.2-20220404.173914-srg.jar%23294!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:forgeendertech:coremod,xf:fml:sereneseasons:temperature_transformer,re:classloading,pl:accesstransformer:B,xf:fml:forgeendertech:coremod,xf:fml:sereneseasons:temperature_transformer,pl:mixin:APP:alternate_current.mixins.json:ServerLevelMixin,pl:mixin:APP:biomemakeover-common.mixins.json:multipart.ServerLevelMixin,pl:mixin:APP:blueprint.mixins.json:ServerLevelMixin,pl:mixin:APP:crafttweaker.mixins.json:common.transform.world.level.MixinServerLevel,pl:mixin:APP:enhancedcelestials.mixins.json:MixinServerWorld,pl:mixin:APP:immersive_weathering.mixins.json:ServerLevellMixin,pl:mixin:APP:immersive_weathering.mixins.json:ServerLevelSnowMixin,pl:mixin:APP:immersiveengineering.mixins.json:coremods.ServerWorldMixin,pl:mixin:APP:radium.mixins.json:ai.nearby_entity_tracking.ServerWorldAccessor,pl:mixin:APP:radium.mixins.json:alloc.chunk_random.ServerWorldMixin,pl:mixin:APP:radium.mixins.json:chunk.entity_class_groups.ServerWorldAccessor,pl:mixin:APP:radium.mixins.json:entity.inactive_navigations.ServerWorldMixin,pl:mixin:APP:radium.mixins.json:profiler.ServerWorldMixin,pl:mixin:APP:snowrealmagic.mixins.json:ServerLevelMixin,pl:mixin:APP:starlight.mixins.json:common.world.ServerWorldMixin,pl:mixin:APP:supplementaries.mixins.json:ServerLevelMixin,pl:mixin:A}
        at net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:902) ~[server-1.18.2-20220404.173914-srg.jar%23294!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer,pl:mixin:APP:elementalcraft.mixins.json:MixinMinecraftServer,pl:mixin:APP:globaldataandresourcepacks.mixins.json:PackConfigurationMixin,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:A}
        at net.minecraft.server.dedicated.DedicatedServer.m_5703_(DedicatedServer.java:326) ~[server-1.18.2-20220404.173914-srg.jar%23294!/:?] {re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
        at net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:842) ~[server-1.18.2-20220404.173914-srg.jar%23294!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer,pl:mixin:APP:elementalcraft.mixins.json:MixinMinecraftServer,pl:mixin:APP:globaldataandresourcepacks.mixins.json:PackConfigurationMixin,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:A}
        at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:685) ~[server-1.18.2-20220404.173914-srg.jar%23294!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer,pl:mixin:APP:elementalcraft.mixins.json:MixinMinecraftServer,pl:mixin:APP:globaldataandresourcepacks.mixins.json:PackConfigurationMixin,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:A}
        at net.minecraft.server.MinecraftServer.m_177918_(MinecraftServer.java:261) ~[server-1.18.2-20220404.173914-srg.jar%23294!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer,pl:mixin:APP:elementalcraft.mixins.json:MixinMinecraftServer,pl:mixin:APP:globaldataandresourcepacks.mixins.json:PackConfigurationMixin,pl:mixin:APP:kubejs-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:kubejs-common.mixins.json:inject_resources.MinecraftServerMixin,pl:mixin:A}
        at java.lang.Thread.run(Thread.java:833) [?:?] {re:mixin}
```

```
A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
        at net.minecraft.world.level.block.state.StateHolder.m_61143_(StateHolder.java:98) ~[server-1.18.2-20220404.173914-srg.jar%23294!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:ferritecore.fastmap.mixin.json:FastMapStateHolderMixin,pl:mixin:A}
        at com.ordana.immersive_weathering.block_growth.hardcoded.CampfireGrowth.tryGrowing(CampfireGrowth.java:35) ~[immersive_weathering-1.18.2-1.2.10.jar%23168!/:1.18.2-1.2.10] {re:classloading}
        at com.ordana.immersive_weathering.block_growth.BlockGrowthHandler.tickBlock(BlockGrowthHandler.java:60) ~[immersive_weathering-1.18.2-1.2.10.jar%23168!/:1.18.2-1.2.10] {re:mixin,re:classloading}
        at net.minecraft.server.level.ServerLevel.handler$zmn000$callTick(ServerLevel.java:4237) ~[server-1.18.2-20220404.173914-srg.jar%23294!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:forgeendertech:coremod,xf:fml:sereneseasons:temperature_transformer,re:classloading,pl:accesstransformer:B,xf:fml:forgeendertech:coremod,xf:fml:sereneseasons:temperature_transformer,pl:mixin:APP:alternate_current.mixins.json:ServerLevelMixin,pl:mixin:APP:biomemakeover-common.mixins.json:multipart.ServerLevelMixin,pl:mixin:APP:blueprint.mixins.json:ServerLevelMixin,pl:mixin:APP:crafttweaker.mixins.json:common.transform.world.level.MixinServerLevel,pl:mixin:APP:enhancedcelestials.mixins.json:MixinServerWorld,pl:mixin:APP:immersive_weathering.mixins.json:ServerLevellMixin,pl:mixin:APP:immersive_weathering.mixins.json:ServerLevelSnowMixin,pl:mixin:APP:immersiveengineering.mixins.json:coremods.ServerWorldMixin,pl:mixin:APP:radium.mixins.json:ai.nearby_entity_tracking.ServerWorldAccessor,pl:mixin:APP:radium.mixins.json:alloc.chunk_random.ServerWorldMixin,pl:mixin:APP:radium.mixins.json:chunk.entity_class_groups.ServerWorldAccessor,pl:mixin:APP:radium.mixins.json:entity.inactive_navigations.ServerWorldMixin,pl:mixin:APP:radium.mixins.json:profiler.ServerWorldMixin,pl:mixin:APP:snowrealmagic.mixins.json:ServerLevelMixin,pl:mixin:APP:starlight.mixins.json:common.world.ServerWorldMixin,pl:mixin:APP:supplementaries.mixins.json:ServerLevelMixin,pl:mixin:A}
        at net.minecraft.server.level.ServerLevel.m_8714_(ServerLevel.java:443) ~[server-1.18.2-20220404.173914-srg.jar%23294!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:forgeendertech:coremod,xf:fml:sereneseasons:temperature_transformer,re:classloading,pl:accesstransformer:B,xf:fml:forgeendertech:coremod,xf:fml:sereneseasons:temperature_transformer,pl:mixin:APP:alternate_current.mixins.json:ServerLevelMixin,pl:mixin:APP:biomemakeover-common.mixins.json:multipart.ServerLevelMixin,pl:mixin:APP:blueprint.mixins.json:ServerLevelMixin,pl:mixin:APP:crafttweaker.mixins.json:common.transform.world.level.MixinServerLevel,pl:mixin:APP:enhancedcelestials.mixins.json:MixinServerWorld,pl:mixin:APP:immersive_weathering.mixins.json:ServerLevellMixin,pl:mixin:APP:immersive_weathering.mixins.json:ServerLevelSnowMixin,pl:mixin:APP:immersiveengineering.mixins.json:coremods.ServerWorldMixin,pl:mixin:APP:radium.mixins.json:ai.nearby_entity_tracking.ServerWorldAccessor,pl:mixin:APP:radium.mixins.json:alloc.chunk_random.ServerWorldMixin,pl:mixin:APP:radium.mixins.json:chunk.entity_class_groups.ServerWorldAccessor,pl:mixin:APP:radium.mixins.json:entity.inactive_navigations.ServerWorldMixin,pl:mixin:APP:radium.mixins.json:profiler.ServerWorldMixin,pl:mixin:APP:snowrealmagic.mixins.json:ServerLevelMixin,pl:mixin:APP:starlight.mixins.json:common.world.ServerWorldMixin,pl:mixin:APP:supplementaries.mixins.json:ServerLevelMixin,pl:mixin:A}
        at net.minecraft.server.level.ServerChunkCache.m_8490_(ServerChunkCache.java:359) ~[server-1.18.2-20220404.173914-srg.jar%23294!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:radium.mixins.json:alloc.chunk_ticking.ServerChunkManagerMixin,pl:mixin:APP:radium.mixins.json:world.chunk_access.ServerChunkManagerMixin,pl:mixin:APP:smoothchunk.mixins.json:ServerChunkCacheMixin,pl:mixin:A}
        at net.minecraft.server.level.ServerChunkCache.m_201698_(ServerChunkCache.java:309) ~[server-1.18.2-20220404.173914-srg.jar%23294!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:radium.mixins.json:alloc.chunk_ticking.ServerChunkManagerMixin,pl:mixin:APP:radium.mixins.json:world.chunk_access.ServerChunkManagerMixin,pl:mixin:APP:smoothchunk.mixins.json:ServerChunkCacheMixin,pl:mixin:A}
        at net.minecraft.server.level.ServerLevel.m_8793_(ServerLevel.java:281) ~[server-1.18.2-20220404.173914-srg.jar%23294!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:forgeendertech:coremod,xf:fml:sereneseasons:temperature_transformer,re:classloading,pl:accesstransformer:B,xf:fml:forgeendertech:coremod,xf:fml:sereneseasons:temperature_transformer,pl:mixin:APP:alternate_current.mixins.json:ServerLevelMixin,pl:mixin:APP:biomemakeover-common.mixins.json:multipart.ServerLevelMixin,pl:mixin:APP:blueprint.mixins.json:ServerLevelMixin,pl:mixin:APP:crafttweaker.mixins.json:common.transform.world.level.MixinServerLevel,pl:mixin:APP:enhancedcelestials.mixins.json:MixinServerWorld,pl:mixin:APP:immersive_weathering.mixins.json:ServerLevellMixin,pl:mixin:APP:immersive_weathering.mixins.json:ServerLevelSnowMixin,pl:mixin:APP:immersiveengineering.mixins.json:coremods.ServerWorldMixin,pl:mixin:APP:radium.mixins.json:ai.nearby_entity_tracking.ServerWorldAccessor,pl:mixin:APP:radium.mixins.json:alloc.chunk_random.ServerWorldMixin,pl:mixin:APP:radium.mixins.json:chunk.entity_class_groups.ServerWorldAccessor,pl:mixin:APP:radium.mixins.json:entity.inactive_navigations.ServerWorldMixin,pl:mixin:APP:radium.mixins.json:profiler.ServerWorldMixin,pl:mixin:APP:snowrealmagic.mixins.json:ServerLevelMixin,pl:mixin:APP:starlight.mixins.json:common.world.ServerWorldMixin,pl:mixin:APP:supplementaries.mixins.json:ServerLevelMixin,pl:mixin:A}
```

**Comment by Cheaterpaul** [comment]
Yes it is fixed in the 0.6.1 version https://www.curseforge.com/minecraft/mc-mods/werewolves-become-a-beast/files/3803356
